docs/index.html

```diff
@@ -38,11 +38,11 @@
     </style>
 </head>
 <body>
-    <section class="hero is-medium is-warning is-bold">
+    <section class="hero is-medium is-light is-bold">
         <div class="hero-body">
             <div class="container has-text-centered">
                 <h1 class="title">
-                    DeNA Codelabs
+                    【ステージング環境】DeNA Codelabs
```

Comment:
ここはaddしてはいけない箇所ですので、元に戻してください。
```suggestion
                    DeNA Codelabs
```